cmd/kubeadm/app/constants/constants.go

```diff
@@ -138,6 +139,9 @@ var (
 	// DefaultTokenUsages specifies the default functions a token will get
 	DefaultTokenUsages = []string{"signing", "authentication"}
 
+	// DefaultTokenGroups specifies the default groups a token will get
+	DefaultTokenGroups = []string{bootstrapapi.BootstrapDefaultGroup}
```

Comment:
See `NodeBootstrapTokenAuthGroup` above. Should use that one; and it should be changed to something like:
 - `system:bootstrappers:kubeadm:node`
 - `system:bootstrappers:kubeadm:default-node`

@ericchiang @liggitt @mikedanese any preference here?